### DISCUSSION TITLE
Getting Path to Current Project

### DISCUSSION THREAD

**Discussion by Martyx00** [rizinorg/rizin#1002] [Q&A] [answered]
Hello,
I wanted to ask whether there is a command which can be used to get full OS path to the currently opened project file.

Thanks

**Comment by ret2libc** [comment]
I think there isn't... @thestr4ng3r ? It would make sense actually to have a way to get information about currently opened project. WDYT?

**Comment by thestr4ng3r** [comment, marked as answer]
There is not really something like a "currently opened" project file. The session is restored from a project file and then it becomes completely independent of that file.
But you can determine the (maybe relative) path of the last saved or loaded file using `e prj.file` and that is also the default path saved to when using `Ps` without arguments.

**Reply by Martyx00** [reply]
I think that should be sufficient for my use case. Thanks!
